apps/astarte_trigger_engine/lib/astarte_trigger_engine/config.ex

```diff
@@ -21,32 +21,95 @@ defmodule Astarte.TriggerEngine.Config do
-  """
-  def events_exchange_name do
-    Application.get_env(:astarte_trigger_engine, :amqp_events_exchange_name)
-  end
+  use Skogsra
 
-  @doc """
-  Returns the AMQP queue name created by the events consumer
-  """
-  def events_queue_name do
-    Application.get_env(:astarte_trigger_engine, :amqp_events_queue_name)
-  end
+  @envdoc "Host for the AMQP consumer connection"
+  app_env :amqp_consumer_host, :astarte_trigger_engine, :amqp_consumer_host,
+    os_env: "TRIGGER_ENGINE_AMQP_CONSUMER_HOST",
+    type: :binary,
+    default: "localhost"
+
+  @envdoc "Port for the AMQP consumer connection"
+  app_env :amqp_consumer_port, :astarte_trigger_engine, :amqp_consumer_port,
+    os_env: "TRIGGER_ENGINE_AMQP_CONSUMER_PORT",
+    type: :integer,
+    default: 5672
+
+  @envdoc "Username for the AMQP consumer connection"
+  app_env :amqp_consumer_username, :astarte_trigger_engine, :amqp_consumer_username,
+    os_env: "TRIGGER_ENGINE_AMQP_CONSUMER_USERNAME",
+    type: :binary,
+    default: "guest"
+
+  @envdoc "Password for the AMQP consumer connection"
+  app_env :amqp_consumer_password, :astarte_trigger_engine, :amqp_consumer_password,
+    os_env: "TRIGGER_ENGINE_AMQP_CONSUMER_PASSWORD",
+    type: :binary,
+    default: "guest"
+
+  @envdoc "Virtual host for the AMQP consumer connection"
+  app_env :amqp_consumer_virtual_host, :astarte_trigger_engine, :amqp_consumer_virtual_host,
+    os_env: "TRIGGER_ENGINE_AMQP_CONSUMER_VIRTUAL_HOST",
+    type: :binary,
+    default: "/"
+
+  @envdoc "The name of the AMQP queue created by the events consumer"
+  app_env :events_queue_name, :astarte_trigger_engine, :amqp_events_queue_name,
+    os_env: "TRIGGER_ENGINE_AMQP_EVENTS_QUEUE_NAME",
+    type: :binary,
+    default: "astarte_events"
+
+  @envdoc "The name of the exchange on which events are published"
+  app_env :events_exchange_name, :astarte_trigger_engine, :amqp_events_exchange_name,
+    os_env: "TRIGGER_ENGINE_AMQP_EVENTS_EXCHANGE_NAME",
+    type: :binary,
+    default: "astarte_events"
+
+  @envdoc "The routing_key used to bind to TriggerEngine specific events"
+  app_env :events_routing_key, :astarte_trigger_engine, :amqp_events_routing_key,
+    os_env: "TRIGGER_ENGINE_AMQP_EVENTS_ROUTING_KEY",
+    type: :binary,
+    default: "trigger_engine"
+
+  @envdoc "A comma separated list of <host>:<port> values of accessible Cassandra nodes"
+  app_env :cassandra_nodes, :astarte_data_access, :cassandra_nodes,
+    os_env: "ASTARTE_CASSANDRA_NODES",
+    type: :binary,
+    default: "localhost:9042"
+
+  @envdoc "A list of host values of accessible Cassandra nodes formatted in the Xandra format"
+  app_env :xandra_nodes, :astarte_data_access, :xandra_nodes,
+    os_env: "ASTARTE_CASSANDRA_NODES",
+    type: XandraNodes,
+    default: ["localhost"]
+
+  @envdoc "A list of {host, port} values of accessible Cassandra nodes in a cqex compliant format"
+  app_env :cqex_cassandra_nodes, :astarte_data_access, :cqex_cassandra_nodes,
```

Comment:
Call it `:cqex_nodes` also here (if we don't have "cassandra" in `:xandra_nodes` it makes sense doing so for consistency)